site/docs/foundations/data-visualization/color-and-pattern.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Color and pattern
+layout: DetailTechnical
+---
+
+Color is a valuable tool that helps effectively present data in your visualization. Using color correctly guides users’ attention and supports their understanding and interpretation of data.
+
+## Color usage
+
+Consider the purpose and impact of color in your chart. Color can be used to:
+
+**Show relationships**
+Use similar colors to imply that items are related. Contrasting colors, however, show that items are from distinct categories.
+
+**Represent data values**
+Color helps to give a clearer indication of range and how data values are distributed; for instance, highs and lows.
+
+**Highlight key elements**
+Use color to direct your audience's attention towards the main element or emphasize data that supports the story you’re communicating.
+
+**Indicate meaning**
+Some colors are strongly associated with certain objects, concepts or emotions. Leveraging these associations helps you communicate more efficiently and improves comprehension of your chart.
+
+<Image
```

Comment:
I'm not sure what this is illustrating, it's a little vague. Perhaps if we included a Chart with positive, negative, and neutral colors we could illustrate color having a strong association with a concept or emotion.

Reply:
Thank you! Could we add a legend to explain the colors and possible update the labels to make it a little clearer too?